features/sample.feature

```diff
@@ -15,15 +15,18 @@ Feature: Hacker News REST API validation
     Then the response status should be "200"
     And the JSON response root should be array
     When I grab "$[0]" as "id"
+    And I save the "$[0]" as "id"
```

Comment:
Precisely, but I had added the additional steps as to not break existing expectations. However, I've reworked the "grab" functionality a bit, such that it functions as you described.